release/tools/schema_gen.sh

```diff
@@ -101,7 +101,7 @@ case `whoami` in
     gangaat )
         export LOAD_PACKAGES='GangaTest:GangaAtlas' ;;
     gangalb )
-        export LOAD_PACKAGES='GangaTest:GangaLHCb' ;;
+        export LOAD_PACKAGES='GangaTest:GangaDirac:GangaLHCb' ;;
```

Comment:
Sure, I will add it now. Again, I'm pretty sure that it's covered by the `LHCb.ini` file so there's no real change here.